templates/rhsm-conduit.yaml

```diff
@@ -329,41 +329,3 @@ objects:
         targetPort: metrics-port
     selector:
       deploymentconfig: rhsm-conduit
```

Comment:
Can this whole file be deleted?

Reply:
This file needs to stay on the main branch at least until this ClowdApp is successfully deploying to stage/the MR lands in app-interface.  There's a whole card in the epic dedicated to deleting these template files that I'm going to do at the end to avoid the complexity of the orders PRs & MRs need to be merged in.

Reply:
ahh makes sense!